docs/development.md

```diff
@@ -55,10 +55,11 @@ To run your plugin in development, you will need to configure a GitHub App to de
 1. [Create a new GitHub App](https://github.com/settings/apps/new) with:
     - **Webhook URL**: Set to `https://example.com/` and we'll update it in a minute.
     - **Webhook Secret:** `development`
-    - **Permissions & events** needed will depend on how you use the bot, but for development it may be easiest to enable everything.
-1. Download the private key and move it to the project directory
-1. Edit `.env` and set `APP_ID` to the ID of the app you just created.
-1. Run `$ npm start` to start the server, which will output `Listening on https://yourname.localtunnel.me`;
+    - **Permissions & events** can be found in the next tab over. What you need to choose will depend on what data you want the bot to have access to. Note: if, for example, you only enable issue events, you will not be able to listen on pull request webhooks with your bot. However, for development we recommend enabling everything.
```

Comment:
I'm +1 for being careful with the use of **bot**. I've described Probot to others and their immediate assumption is that its a chatbot framework for use on GitHub (which it isn't really).